addons/website/static/src/components/fullscreen_indication/fullscreen_indication.js

```diff
@@ -30,6 +30,6 @@ export class FullscreenIndication extends Component {
     }
 
     get fullScreenIndicationText() {
-        return markup(sprintf(escape(_t("Press %(key)s to exit full string")), "<span>esc</span>"));
+        return markup(sprintf(escape(_t("Press %(key)s to exit full screen")), "<span>esc</span>"));
```

Comment:
@odoo/rd-security Could you validate the CI here? Forward port of already validated change, which is still safe, just some controlled code in markup